src/assets/Timecard.scss

```diff
@@ -1,3 +1,8 @@
+.grey-border {
```

Comment:
We've moved it into a `common.scss` file and used the GDS variables, but we aren't sure about separating the border and padding out - maybe if we need to use a different value for the padding in future it would make sense to split them out then?